src/lib/elements/forms/inputDigits.svelte

```diff
@@ -24,10 +25,30 @@
         }
     });
 
-    onMount(() => {
+    /**
+     * TODO: Find proper solution.
+     * This spams focus for half a second.
+     * Autofocus works properly if +layout.ts is ssr = false. Seems like Svelte Kit bug.
+     * Using document.hasFocus doesnt help here
+     */
+    async function focusInput(i = 0) {
+        if (i >= 10) {
+            return;
+        }
+
         if (element && autofocus) {
             element.querySelector('input')?.focus();
         }
+
+        await tick();
+
+        setTimeout(() => {
+            focusInput(i + 1);
+        }, 50);
+    }
+
+    onMount(() => {
+        focusInput();
```

Comment:
can you check if this would work?

```js
let interval = setInterval(() => {
    const element = element.querySelector('input');
    if (element) {
        element.focus();
        clearInterval(interval);
    }
}, 50);

Reply:
Doesnt work. `if (element) {` is true on first iteration. Trust me, no elegant solution here. All values in HTML seem to say what it should, but then Svelte Kit must interfere to unfocus. And it starts to work if `ssr = true` in `layout.page`.
I even tried `document.hasFocus` and that was true as expected, but then few ticks later it became false.